protobuf/bess_msg.proto

```diff
@@ -151,6 +151,18 @@ message ListWorkersResponse {
   repeated WorkerStatus workers_status = 2;
 }
 
+message GetLinkStatusRequest {
```

Comment:
Ok.  I could go either way on this.  In a narrow sense it is self descriptive but a) it's nice to have everything documented for consistency so no guesswork is required and the meaning of Link and Status aren't actually totally obvious in the BESS context.